Gemfile

```diff
@@ -0,0 +1,11 @@
+ruby "~> 2.3.0"
```

Comment:
Gemfile not needed since it is same as microsoft/terraform-test

Reply:
The Gemfile's are actually different than the base microsoft/terraform-test image and needed for local execution.

Reply:
We can easily update the microsoft/terraform-test image.  Is there something special about this module in regards to testing where only this module needs something unique?